blogs/scripts/scripts.js

```diff
@@ -504,6 +504,30 @@ async function loadLazy(doc) {
   if (templateName) {
     await loadTemplate(doc, templateName);
   }
+  // add back office tags to head
+  const backoffice = getMetadata('back-office-tags');
```

Comment:
I'd suggest extracting this to its own function. Also, I think we need to validate these tags as well, so I'd probably move this into the delayed.js as a result since that validation can be slow.